src/mediatype-icon.ts

```diff
@@ -32,22 +32,20 @@ export class MediatypeIcon extends LitElement {
   }
 
   render() {
-    if (!this.mediatype) {
+    if (!this.displayMediatype) {
       return html``;
     }
 
+    const toDisplayMediatype = mediatypeConfig[this.displayMediatype];
```

Comment:
Suggestion.. since this is the config, I would name it as such. The display media type is the thing you're using to look up the config.

Also, If you do this just below, you can get rid of those `ifDefined()` in the HTML since it will always be defined:
```
if (!toDisplayMediaType) { return html`` }
```

Reply:
updated 🙇